.env

```diff
@@ -1 +1,4 @@
 PORT=3001
+
+REACT_APP_BACKEND_HOST="http://localhost:3000"
+REACT_APP_BACKEND_VERSION="> 0.0.0-90"
```

Comment:
As a work-around for now, I can symlink `package.json` to `src/config.json`. Any downsides to symlinks in git?